lib/framework/action.dart

```diff
@@ -222,7 +222,16 @@ class FileUploadAction extends EnsembleAction {
   String? overMaxFileSizeMessage;
 }
 
+class CopyToClipboardAction extends EnsembleAction {
+  CopyToClipboardAction({
+    this.value,
+    this.onComplete,
+  });
+  String? value;
+  EnsembleAction? onComplete;
```

Comment:
use `onSuccess` instead. Add `onFailure` calllback in case the value is empty or there's an error (don't throw the error like you did below). The user can show the toast error if they want.

Reply:
i added these callback